pkg/server/commands/list_objects.go

```diff
@@ -213,14 +213,14 @@ func (q *ListObjectsQuery) Execute(
 			}, nil
 
 		case objectID, ok := <-resultsChan:
-			if !ok {
+			if !ok { //channel was closed and no new values are available
```

Comment:
This comment and the one below seem redundant. This is the behavior of the language. It should go without documentation.

Reply:
I understand that it's part of the language but it's not super obvious to me. I don't think it hurts to have it

Reply:
How about a different variable name then?

`objectID, closed := <-resultsChan`